library/cartridge_check_instance_started.py

```diff
@@ -4,32 +4,93 @@
 from ansible.module_utils.helpers import ModuleRes, CartridgeException
 from ansible.module_utils.helpers import get_control_console
 
-
 argument_spec = {
     'control_sock': {'required': True, 'type': 'str'},
     'stateboard': {'required': True, 'type': 'bool'},
+    'expected_state': {'required': False, 'type': 'list', 'default': ['Unconfigured', 'RolesConfigured']},
+    'check_buckets_discovery': {'required': False, 'type': 'bool', 'default': False},
```

Comment:
`check_buckets_discovering`?
`check_buckets_are_discoveed`?

Reply:
Why not, fixed